src/WPF/WPF.Viewer/Samples/UtilityNetwork/CreateLoadReport/readme.md

```diff
@@ -0,0 +1,60 @@
+# Create load report
+
+Demonstrates the creation of a simple electric distribution report. It traces downstream from a given point and adds up the count of customers and total load per phase.
+
+![Image of load report](CreateLoadReport.jpg)
+
+## Use case
+
+You can use a load report to display the customers per phase as well as the load per phase based on a chosen starting point in a utility network. Load reports are used for electric load restoration and balancing.
+
+## How to use the sample
+
+Choose phases to be included in the report. Click 'Run Report' to initiate a downstream trace on the network and create a load report. Click 'Reset' to clear the phases and start over.
+
+## How it works
+
+1. Create and load a `UtilityNetwork` with a feature service URL, then get an asset type, tier, network attributes, and category by their names.
+2. Create a `UtilityElement` from the asset type to use as the starting location for the trace.
+3. Get a base condition from the `UtilityTier.traceConfiguration.traversability.barriers`.
```

Comment:
I think the properties in the readme here should be capitalized as they are in the documentation. 

`UtilityTier.TraceConfiguration.Traversability.Barriers`